sidebarsHyperIndex.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
         // "Guides/subgraph-mPigration",
         "Guides/testing",
         "Guides/navigating-hasura",
+        "Guides/environment-variables",
```

Comment:
Should it maybe in the advanced section?

Reply:
It could but I'm not sure it should